library/build.gradle

```diff
@@ -78,9 +79,11 @@ uploadArchives {
         }
 
         pom.project {
-            name 'Android In-App Billing v3 Library'
+            name 'Android In-App Billing v4 Library'
             packaging 'aar'
-            description 'A lightweight implementation of Android In-app Billing Version 3'
+            description 'A lightweight implementation of the current Android In-app Billing Version 4, This will ' +
+                    'successfully removing the warning on all developer dashboards about Google warning that all apps must use ' +
```

Comment:
can you please remove the 'Google warning' part?

Reply:
ok